src/pages/ReportDetailsPage.tsx

```diff
@@ -90,6 +90,12 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
         return ReportUtils.getVisibleChatMemberAccountIDs(report.reportID ?? '');
     }, [report, isGroupChat]);
 
+    // Get the active chat members by filtering out the pending members with delete action
+    const activeChatMembers = participants.flatMap((accountID) => {
+        const pendingMember = report?.pendingChatMembers?.findLast((member) => member.accountID === accountID.toString());
+        return !pendingMember || pendingMember.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE ? accountID : [];
+    });
```

Comment:
We do not filter out the `accountID` that is not available in the `personalDetails` object. This resulted in https://github.com/Expensify/App/issues/46670